apps/sms/lib/bridge/client.js

```diff
@@ -606,10 +606,11 @@ var debug = {
 
 /**
  * Default response timeout.
+ *
```

Comment:
Used the chance to update bridge libs as well (mainly because DOMError handling has been improved) - tell me if you'd like separate bug for these changes